pages/blog/posts/applicability-json-schema-fundamentals-part-1.md

```diff
@@ -429,6 +429,3 @@ All feedback is welcome. If you have questions or comments, you can find me on t
 - [JSON Schema live playground (ajv) for draft-07 with shareable links](https://jsonschema.dev)
 - [JSON Schema live playground (hyperjump validator) for all drafts (from draft-04)](https://json-schema.hyperjump.io)
 - [Links to RFC specification documents](/specification)
-
-
-Photo by [Heidi Fin](https://unsplash.com/@heidijfin?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)
```

Comment:
This was removed but nothing was added in its place.

Reply:
Just leave this:

Photo by Heidi Fin on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText).

Reply:
Done!